app/controllers/advertisements_controller.rb

```diff
@@ -261,18 +261,18 @@ def get_fallback_campaign(campaign_relation)
     end
 
     # pre-selected fallback
-    chosen_campaign = choose_campaign(fallback_campaign_relation, ignore_budgets: true)
+    campaign = choose_campaign(fallback_campaign_relation, ignore_budgets: true)
```

Comment:
Won't this line prevent the campaign selections below from running?

Reply:
Oh.. possibly.

Reply:
Nope. `fallback_campaign_relation` already has the pre-selected scope from above